relsint.rdf

```diff
@@ -22,22 +22,47 @@
     <dcterms:modified rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2015-05-11</dcterms:modified>
     <dcterms:publisher rdf:resource="http://islandora.ca/"/>
     <rdfs:comment xml:lang="en">Ontology for the Islandora RELS-INT data model, intended primarily to make it possible to expose Islandora-curated RDF predicates via de-reference-able URIs.</rdfs:comment>
-    <owl:versionInfo>v1/2015/05/11</owl:versionInfo>
+    <owl:versionInfo>v1/2016/02/22</owl:versionInfo>
   </rdf:Description>
   
-  <!-- http://islandora.ca/ontology/relsint#height -->
-  <rdf:Property rdf:about="http://islandora.ca/ontology/relsint#height">
-    <rdfs:label xml:lang="en">height</rdfs:label>
+  <!-- http://islandora.ca/ontology/relsint#height_literal -->
+  <rdf:Property rdf:about="http://islandora.ca/ontology/relsint#height_literal">
+    <rdfs:label xml:lang="en">height literal</rdfs:label>
     <rdfs:comment xml:lang="en">The height of an image</rdfs:comment>
```

Comment:
I don't think xml attributes for stuff like that are valid RDF (because they can't be expressed as a true triple), but please correct me if you know otherwise.